configs/llama-v2-7b/README.md

```diff
@@ -42,40 +42,40 @@ To use `prompt flow` in VS Code, please refer to this [Quick Start](https://micr
 
 ## Remote Development
 ### Prerequisites
-1. To run the model fine-tuning in your remote Azure Container App Environment, you need to make sure your subscription have enough GPU capacity amount. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the capacity amount required for your application. [Learn More about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
+1. To run the model fine-tuning in your remote Azure Container App Environment, your subscription has enough GPU capacity. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the required capacity for your application. [Get More Info about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
```

Comment:
Add 'make sure' after the comma, i.e. ....Environment, make sure your subscription......

Reply:
Updated